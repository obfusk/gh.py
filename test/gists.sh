#!/bin/bash
"$PY" gh.py gists obfusk \
  | jq '[.[]|select(.desc|contains("pry"))]'
