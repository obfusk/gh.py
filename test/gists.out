[
  {
    "link": "https://gist.github.com/208597ccc64bf9b436ed",
    "desc": "python equivalent of ruby's binding.pry"
  }
]
